src/views/ArticleCreation.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { Redirect } from 'react-router-dom'
+import { Box, Typography } from '@material-ui/core'
+import Articles from '../modules/Articles'
+import ArticleCreationForm from '../components/ArticleCreationForm'
+import SuccessSnackbar from '../components/popups/SuccessSnackbar'
+import useCommonStyles from '../theme/useCommonStyles'
+
+const newArticle = {
+  title: '',
+  body: '',
+}
```

Comment:
I think put it in the state, though I don't insist, that would be more readable for me. Or rename to emptyArticle

Reply:
Fixed